pages/linux/dpkg.md

```diff
@@ -21,3 +21,7 @@
 - Find out which package owns a file:
 
 `dpkg -S {{file_name}}`
+
+- List contents of local package file:
+
+`dpkg -c {{/path/to/file}}`
```

Comment:
Remove the prefixing slash from here. 👍

Reply:
For the sake of consistency, should it be removed from the first entry as well? i.e. `dpkg -i` example (looks like it was missed during #947)